runelite-mixins/src/main/java/net/runelite/mixins/RSClientMixin.java

```diff
@@ -504,14 +504,8 @@ public int getSkillExperience(Skill skill)
 
 		if (skill == Skill.OVERALL)
 		{
-			int totalExperience = 0;
-
-			for (int experience : experiences)
-			{
-				totalExperience += experience;
-			}
-
-			return totalExperience;
+			logger.debug("getSkillExperience called for {}!", skill);
```

Comment:
This debug log is probably just spam

Reply:
This would indicate incorrect usage, so it should be logged.

Reply:
Oh i see, with your other reply it makes sense. Maybe I would then make this method not accept Overall, e.g do assert

Reply:
I'm not that confident that it won't break something currently if we did that.

Reply:
Well ok